### ISSUE TITLE
Library view / bookshelf, cover images: not alt="" role="presentation" (or some other text description, like title? ... maybe redundant with other labels though)

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#586]

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/568

**Comment by danielweck** [comment]
Further testing + analysis: when a publication has no cover image, keyboard-tabbing into the auto-generated cover visuals (i.e. CSS gradient background colour) results in the screen reader emitting an accessible description based on the text overlay inside the cover image rectangle (i.e. title + author). Conversely, when a publication has a cover image, there is no accessible description (just an announcement of JPEG image, for example). This should be consistent, so I would recommend adding the exact same text (title + author) as `alt` image attribute (or `title` on the parent link, and `alt="" role="presentation"` on the image itself).

But, this creates redundancy with the menu button. So, I suggest reducing the verbosity of the accessible label for the menu button, on the basis that the cover image (or its fake image placeholder) is reached first when keyboard-tabbing. Users will quickly "see" that the menu button always follows the cover image, allowing them to quickly move forward (or backward) when using keyboard navigation to browser the bookshelf, without redundant annoying announcements.

**Comment by Scarsniik** [comment]
Fixed by #633
